activerecord/test/cases/arel/insert_manager_test.rb

```diff
@@ -11,19 +11,18 @@ class InsertManagerTest < Arel::Spec
     end
 
     describe "insert" do
-      it "can create a Values node" do
+      it "can create a ValuesList node" do
         manager = Arel::InsertManager.new
-        values  = manager.create_values %w{ a b }, %w{ c d }
+        values  = manager.create_values_list([%w{ a b }, %w{ c d }])
 
-        assert_kind_of Arel::Nodes::Values, values
-        assert_equal %w{ a b }, values.left
-        assert_equal %w{ c d }, values.right
+        assert_kind_of Arel::Nodes::ValuesList, values
+        assert_equal [%w{ a b }, %w{ c d }], values.rows
       end
 
       it "allows sql literals" do
         manager = Arel::InsertManager.new
         manager.into Table.new(:users)
-        manager.values = manager.create_values [Arel.sql("*")], %w{ a }
+        manager.values = manager.create_values([Arel.sql("*")])
```

Comment:
We can maintain the existing test here? Or it was already unwarranted?

Reply:
It could be maintained by `def create_values(values, _columns = nil)` conservatively, but the `columns` arg is not used even if in `Arel::Nodes::Values`, and the `create_values` private API is not directly used even in the code base.
I think it is less worth to maintain the unused args for the private API.